# Pulses per second (lowest possible speed is 20 seconds for one wheel revolution)
uint16 speed
# Enumeration rotDirection is used
uint8 direction
# Timestamp for measurement
uint32 when
# Odometer when direction changed
uint32 dist
# Absolute distance travelled
uint32 dist_abs
